### DISCUSSION TITLE
Is lazypredict officially abandoned?

### DISCUSSION THREAD

**Discussion by JSchoeck** [shankarpandala/lazypredict#443] [General]
Hey @shankarpandala 
are you still considering continuing the work on lazypredict?
As there are several open pull-requests and the code does not work without fixes with current dependencies, it would be great if you could either get someone else to continue the code, or accept PRs of people who fix open issues.

lazypredict is a great concept, keep it alive! :)

**Comment by BradKML** [comment]
He is probably busy but since he takes your pull request, maybe people can share responsibility over this repo (forking maybe?). Hope that people can start modding this repo again

**Reply by BradKML** [reply]
P.S. I saw this repo that seems to be active, so might be good to back it https://github.com/nityansuman/lazypredict-nightly
